test/Microsoft.AspNet.Identity.EntityFramework.Test/DefaultPocoTest.cs

```diff
@@ -63,6 +65,152 @@ public void DropDatabaseStart()
             IdentityResultAssert.IsSuccess(await userManager.DeleteAsync(user));
         }
 
+        [Fact]
+        public async Task CanIncludeUserClaimsTest()
+        {
+            // Arrange
+            CreateContext(true);
+            var builder = new ApplicationBuilder(CallContextServiceLocator.Locator.ServiceProvider);
+
+            builder.UseServices(services =>
+            {
+                DbUtil.ConfigureDbServices<IdentityDbContext>(ConnectionString, services);
+                services.AddIdentity<IdentityUser, IdentityRole>().AddEntityFrameworkStores<IdentityDbContext>();
+            });
+
+            var userManager = builder.ApplicationServices.GetRequiredService<UserManager<IdentityUser>>();
+            var dbContext = builder.ApplicationServices.GetRequiredService<IdentityDbContext>();
```

Comment:
However this still doesn't seem right. When will the context get disposed? I assume you copied this pattern from other tests?